osbuild/pipeline.py

```diff
@@ -371,7 +371,11 @@ def __init__(self):
         self.sources: List[Source] = []
 
     def add_pipeline(
-            self, name: str, runner: Optional[str], build: Optional[str] = None, source_epoch: Optional[int] = None
+        self,
+        name: str,
+        runner: Runner,
+        build: Optional[str] = None,
+        source_epoch: Optional[int] = None
```

Comment:
\o/ (btw., this even allows for trailing commas after the last argument, isn't this great?)